src/silly_sprites/opengl/shader_program.cpp

```diff
@@ -0,0 +1,141 @@
+#include "shader_program.hpp"
+#include "utils.hpp"
+
+namespace sly::gl {
+    static constexpr char const* fallback_vertex_source{ R"(
```

Comment:
According to the styleguide, initializations should all use the `=` sign. Also you can use `auto` here.
```suggestion
    static constexpr auto fallback_vertex_source = R"(
```